pkg/operator/k8sutil/pod.go

```diff
@@ -118,6 +118,32 @@ func GetSpecContainerImage(spec v1.PodSpec, name string, initContainer bool) (st
 	}
 	return image.Image, nil
 }
+func AddUnreachableNodeToleration(podSpec *v1.PodSpec, tolerationSeconds int64) {
+	// Node controller will wait 40 seconds by default before mark a node as
+	// unreachable (node.kubernetes.io/unreachable). This function adds a
+	// Toleration of <seconds> in the pod for this
+	// situation. After 40 + n seconds the pod will be scheduled in other node
+	// Only one <toleration> to <unreachable> nodes can be added
+	urToleration := v1.Toleration{Key: "node.kubernetes.io/unreachable",
+		Operator:          "Exists",
+		Effect:            "NoExecute",
+		TolerationSeconds: &tolerationSeconds}
+
+	var replaced bool = false
+
+	for index, item := range podSpec.Tolerations {
+		if item.Key == "node.kubernetes.io/unreachable" {
+			podSpec.Tolerations[index] = urToleration
+			replaced = true
+		}
+		break
+	}
+
+	if !replaced {
+		podSpec.Tolerations = append(podSpec.Tolerations, urToleration)
+	}
+
```

Comment:
nit: remove a blank line after a closing brace

Reply:
done

Reply:
This hasn't been addressed. The blank line is till there.

Reply:
Sorry. Blank lines removed